parsec/core/gui/lang.py

```diff
@@ -20,13 +20,17 @@
 
 
 def format_datetime(dt, full=False, seconds=False):
-    fmt = "L LT"
-    if seconds:
-        fmt = "L LTS"
-    if full:
-        fmt = "LLLL"
-    # The alternative formatter is now the default one since pendulum 2.0.0
-    return dt.in_tz(pendulum.local_timezone()).format(fmt, locale=_current_locale_language)
+    # Handle if it is already a LocalDateTime
+    if isinstance(dt, DateTime):
+        dt = dt.to_local()
+    if _current_locale_language == "fr":
```

Comment:
I would do the opposite: test if lang is `en` and do the 'French style' otherwise. French style tends to be the most commonly used.
I do agree that we only have two locales and it's probably going to stay that way, and it's not correct anyway since it should depend on the time locale, but it's a simple switch.